### DISCUSSION TITLE
What's the highest version of desktop that still works on Windows 7?

### DISCUSSION THREAD

**Discussion by Robodanny** [jgraph/drawio-desktop#1305] [Q&A] [answered]
The latest desktop release (21.2.1) does not run on Windows 7.

DiscardVirtualMemory not found in KERNEL32.dll

![DiscardVirtualMemory](https://user-images.githubusercontent.com/13823662/235313659-82f447f2-debd-42c9-a4a6-96701d1c9a17.png)

I had v20.8.16 running.  What's the highest I can go on Win7?

**Comment by davidjgraph** [comment, marked as answer]
21.1.2 is the last release that supports Windows 7 or 8.

**Reply by Robodanny** [reply]
Thanks for the quick reply and a great program.  draw.io is by far my favorite technical drawing tool!
